src/DynamoCore/Extensions/IExtensionLoader.cs

```diff
@@ -10,7 +10,14 @@ namespace Dynamo.Extensions
     /// </summary>
     public interface IExtensionLoader
     {
+        /// <summary>
+        /// Loads assembly by passed extension path and return it as IExtension
+        /// </summary>
         IExtension Load(string extensionPath);
+
+        /// <summary>
+        /// Returns an enum of IExtension specified by passed extension path
```

Comment:
Extension method for loading assembly from a directory. Returns cref = IExtension.